### PR TITLE
Feature/delete member

### PR DESCRIPTION
**Pull request by mariiaovs**

### CODE REVIEW

File: components/MemberProfile.js [hunk 1]
```diff
@@ -338,32 +427,49 @@ export default function MemberProfile({
...
+        {showModal && modalMode === "delete-account" && (
+          <ConfirmBox
+            onConfirm={handleDeleteAccount}
+            message={deleteAccountMessage}
+          />
+        )}
+      </Modal>
     </>
   );
 }
```
Comment (nichoop) [inline components/MemberProfile.js:475]: You might consider moving some components in the code to another file or folder for better readability.

**Review by nichoop — APPROVED** [review]
Good job! I've just added a small comment.

File: components/MemberProfile.js [hunk 2]
```diff
@@ -338,32 +427,49 @@ export default function MemberProfile({
...
       </Modal>
+      <Modal $top="13.5rem" $open={showModal && modalMode === "delete-account"}>
+        {showModal && modalMode === "delete-account" && (
+          <ConfirmBox
+            onConfirm={handleDeleteAccount}
+            message={deleteAccountMessage}
+          />
+        )}
+      </Modal>
     </>
```
[the remark this answers is not in the document]
Comment (mariiaovs) [reply components/MemberProfile.js:473]: Thank you, we can do it in the next developer user story.

File: pages/api/deleteImage/index.js [hunk 3]
```diff
@@ -9,7 +9,8 @@ cloudinary.config({
 
 export default async function handler(request, response) {
   if (request.method === "DELETE") {
-    const { imageUrl } = request.body;
+    const { profilePhoto: imageUrl } = request.body;
+    console.log(imageUrl);
```
Comment (niehaus-alexander) [inline pages/api/deleteImage/index.js:13]: remove console.logs

**Review by niehaus-alexander — APPROVED** [review]
LGTM!

**Pull request merged**